default.nix

```diff
@@ -167,6 +167,35 @@ in rec {
 
   inherit mkAssets;
 
+  dockerImageConfig = args@{exe, name, internalPort ? 8000, version ? "latest", created ? "now",extraContents ? [], extraPaths ? []}:
+    let
+      appDirSetupScript = nixpkgs.runCommand "appDirSetupScript.sh" {} ''
+        mkdir -p    $out/var/lib/backend
+        ln -sft $out/var/lib/backend '${exe}'/*
+        ${nixpkgs.findutils}/bin/find $out/var/lib/backend
+        '';
+    in {
+      name = name;
+      tag = version;
+      created = created;
+      contents = [ nixpkgs.iana-etc nixpkgs.cacert appDirSetupScript ] ++ extraContents;
+      config = {
+        Env = [
+          ("PATH=" + builtins.concatStringsSep(":")(extraPaths ++ [
```

Comment:
I think parens around `":"` not required

Reply:
This seems necesary, got the following error when testing:

```
error: syntax error, unexpected ':', at /home/fabian/Development/Open/obelisk/default.nix:191:48
```